src/test/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheckTest.java

```diff
@@ -237,9 +237,9 @@ public void testSetMessage() throws Exception {
         checkConfig.addAttribute("minimum", "500");
         checkConfig.addAttribute("message", "someMessage");
 
-        final String[] expected = new String[223];
-        for (int i = 0; i < 223; i++) {
-            expected[i] = i + ": someMessage";
+        final String[] expected = new String[222];
+        for (int i = 0; i < 222; i++) {
+            expected[i] = i + 1 + ": someMessage";
```

Comment:
@rnveach Thanks for the reminder. After some investigation, I find the pitest error is due to these three lines of changes. I notice that there is one VoidMethodCallMutator case that survived, leading to the pitest error. However, I am having some difficulty finding the reason. I just wonder if there is any way to see the actual mutation or any other effective debugging approaches.

Reply:
@li-boxuan Please run pitest and upload the html file for the class. It specifies the mutations done and which ones are surviving. With that report, we can see where the problem area is and how to best help.
The command given above should generate the html report in `target` automatically.

Reply:
It could be the `< 222` and it is changing it to `>= 222` and no tests are failing.
It could be the `+1` and it is changing it to `-1`.
I could be wrong since we can't see the report.

Reply:
Thanks for the hint. The problem is this line https://github.com/checkstyle/checkstyle/pull/6095/files#diff-cbb6a2c93e57326ba07d4d3c84bbcdf7R129. (`removed call to com/puppycrawl/tools/checkstyle/api/AbstractViolationReporter::log → NO_COVERAGE`)

Previously, we report line number 0 when `minimum` is not satisfied for `RegexpMultilineCheck`. However, after my changes, we report line number 1. At the same time, we also have another report message saying that line 1 has matched the regular expression.

Previously (223 error messages):
0: some message
1: some message
2: some message
...

Currently (223 error messages):
1: some message
1: some message
2: some message
...

The duplication is somehow removed and only 222 error messages are reported finally in the test. Therefore, removal of https://github.com/checkstyle/checkstyle/pull/6095/files#diff-cbb6a2c93e57326ba07d4d3c84bbcdf7R129 would not affect the test, leading to the pitest error.

Reply:
@romani ping

Reply:
I think I’ve fixed the issue, however, Travis is now failing due to some jacoco errors which don’t seem to be related to my pr.

`org.apache.maven.plugin.PluginResolutionException: Plugin org.jacoco:jacoco-maven-plugin:0.8.2 or one of its dependencies could not be resolved: Failed to read artifact descriptor for org.jacoco:jacoco-maven-plugin:jar:0.8.2`

I repushed and triggered another build, which did no help.